kms_log/main.tf

```diff
@@ -14,11 +14,17 @@ locals {
 
   lambda_env_variables = merge(local.default_variables, local.alarm_variables)
 
+  lambda_insights = "arn:aws:lambda:${var.region}:580247275435:layer:LambdaInsightsExtension:38"
```

Comment:
Should this whole arn be a variable? Or maybe add a variable for the version number? It looks like the extension is updated every 2-4 months.

Reply:
Agreed -- I would go one further and suggest variable-ize-ing the account number, as well, and making a note of what account it refers to (which is unclear from here).

Reply:
Requested Changes have been implemented!